src/ai/backend/web/proxy.py

```diff
@@ -399,6 +400,25 @@ async def websocket_handler(request, *, is_anonymous=False) -> web.StreamRespons
                 content_type=request.content_type,
                 override_api_version=request_api_version,
             )
+            if proxy_path == "pipeline":
+                aiohttp_session = request.cookies.get("AIOHTTP_SESSION")
+                if not (sso_token := request.headers.get("X-BackendAI-SSO")):
+                    jwt_secret = request.app["config"]["pipeline"]["jwt"]["secret"]
+                    now = datetime.now(tz=timezone(timedelta(hours=9)))
```

Comment:
What does that `timedelta(hours=9)` mean??
Does it mean that this will be expired in 9 hours or something else?

Reply:
This is to set the timezone to KST(UTC+09:00).

Reply:
The token is for external services, such as the pipeline service. (https://github.com/lablup/backend.ai-fasttrack/blob/7b3832163a7fe0debd2c5c0c88614aa6b1c3cf28/config/asgi.py#L61-L86)